flow/designs/tsmc65lp/black_parrot/config_hier.mk

```diff
@@ -0,0 +1,17 @@
+include $(dir $(DESIGN_CONFIG))/config.mk
+
+export FLOW_VARIANT ?= hier_rtlmp
+export SYNTH_HIERARCHICAL = 1
+export MAX_UNGROUP_SIZE = 10000
+export RTLMP_FLOW = True
+#
+# RTL_MP Settings
+export RTLMP_MAX_INST = 30000
+export RTLMP_MIN_INST = 5000
+export RTLMP_MAX_MACRO = 12
+export RTLMP_MIN_MACRO = 4 
```

Comment:
Having to manually tune this for each design isn't great.  We should plan to make rtlmp smarter in its defaults.